glide.lock

```diff
@@ -108,10 +154,12 @@ imports:
   - repository
   - test
 - name: gopkg.in/src-d/framework.v0
-  version: 06c61f8f6a8913f3d4d465bab758ff06c0b3d920
+  version: 9bf13f3e57f5175ba43bc8c4def79a024041a57f
+  repo: https://github.com/smola/framework.git
```

Comment:
leftover in this PR? we should merge first framework and then upload this lock file.